Sources/FoundationEssentials/NotificationCenter/ActorQueueManager.swift

```diff
@@ -0,0 +1,91 @@
+//===----------------------------------------------------------------------===//
```

Comment:
We'll need to add these new files to the CMake build by listing them in the `CMakeLists.txt` files. You can take a look at the existing `CMakeLists.txt` files in `Sources/FoundationEssentials` and `Sources/FoundationEssentials/AttributedString` as an example.

Reply:
Thanks. Done.